storybook/pages/BackupSeedModalPage.qml

```diff
@@ -25,7 +25,6 @@ Item {
     Label {
         anchors.bottom: parent.bottom
         anchors.horizontalCenter: parent.horizontalCenter
-        //text: "Hint: %1".arg(popup.mnemonic)
```

Comment:
Can it be removed?

Reply:
What exactly? I just removed the extra, commented-out stuff

Reply:
Oops, my bad. I don't know why I thought you just commented a line and left it there.